requirements.txt

```diff
@@ -0,0 +1,12 @@
+ansible==7.4.0
+ansible-core==2.14.4
```

Comment:
I believe requiring ansible-core separately is unnecessary, as it's already included in and installed with the current version of ansible

Reply:
The requirements.txt was auto generated by running `pip freeze` after installing the latest available ansible version. There are other ways to generate the requirements.txt (including manual or template based ones), but i think to get started here it is the easiest to just for with a currently stable environment including all python dependencies.